modules/bookList.js

```diff
@@ -0,0 +1,60 @@
+// // Book Class: Represents a Book
+
+// getBooks(() => {
+//   let books;
+//   if (localStorage.getItem("books") === null) {
+//     books = [];
+//   } else {
+//     books = JSON.parse(localStorage.getItem("books"));
+//   }
+
+//   return books;
+// });
+
+// export default addBook((book) => {
+//   const books = getBooks();
+//   books.push(book);
+//   localStorage.setItem("books", JSON.stringify(books));
+// });
+// export default removeBook((author) => {
+//   const books = getBooks();
+
+//   books.forEach((book, index) => {
+//     if (book.author === author) {
+//       books.splice(index, 1);
+//     }
+//   });
+
+//   localStorage.setItem("books", JSON.stringify(books));
+// });
```

Comment:
- [ ] [**Optional**] Kindly remove the commented pieces of code because they make your work look messy.

Reply:
oh, okay, sure thanks, I really forgot to do it. thank you